### ISSUE TITLE
[Package request] install

### ISSUE THREAD

**Issue by hmaarrfk** [conda-forge/staged-recipes#16154]
# The requested package `install`

- _Package name_: install
- _Version (If a specific one is required)_:
- _URL_: https://pypi.org/project/install/
- _Available via (GitHub Releases, PyPI, etc.)_: pypi
- _Additional comments_: Seems to be a pip dependency of tensorflow-gpu

# Before submitting
- [x] Verify that the package does not already exist (Also check with _hyphens_ vs. _underscores_ and added identifiers like `python-` and `r-`).
- [x] Check that no previous [issue](https://github.com/conda-forge/staged-recipes/issues) exists and that no [PR](https://github.com/conda-forge/staged-recipes/pulls) is currently open.

**Comment by jakirkham** [comment]
This might be another case where we need to `python-` prefix something

cc @conda-forge/core

**Comment by jakirkham** [comment]
The repo is called [`pip-install`]( https://github.com/eugenekolo/pip-install ). So maybe that is another option (if that is not to confusing 😟)

**Comment by beckermr** [comment]
Ummmmm.....

```
conda install pip-install
```

**Comment by hmaarrfk** [comment]
is it the same package. i guess that works!

**Comment by jakirkham** [comment]
I don't think that was actually added that way. At least I'm not seeing it in our repos or packages. Think Matt was merely pointing out that would be an awkward naming choice

**Comment by beckermr** [comment]
I think `pypa-install` is the best choice but we probably need to follow what we did for `build` or go back and change `build`

**Comment by beckermr** [comment]
@jezdez any comments here?

**Comment by jakirkham** [comment]
This particular discussion reminds me of the namespace discussion ( https://github.com/conda/conda/issues/6939 ). Obviously that is a longer term thing and we will likely want a solution for this package before that is solved. More just noting this is one more thing that would be solved by that.

**Comment by danielnachun** [comment]
The URL linked is no longer valid, and we have long since packaged tensorflow-gpu, so I think we can close this out.
